rmf_visualization_navgraphs/src/NavGraphVisualizer.cpp

```diff
@@ -203,7 +203,8 @@ auto NavGraphVisualizer::FleetNavGraph::update_lane_states(
   std::optional<std::string> map_name_filter) -> std::vector<Marker::ConstSharedPtr>
 {
   if (!traffic_graph.has_value())
-    return {};
+    return {}
```

Comment:
This diff looks a bit weird. I think if we wrap that if inside braces, the semicolon will not move into a newline.